### ISSUE TITLE
[Accessibility Docs] Accordion

### ISSUE THREAD

**Issue by sowasred2012** [canonical-web-and-design/vanilla-framework#4037]
Using [this template](https://docs.google.com/document/d/1IwFMfJrTfJPXDcVJfLWXYkw-pjgWhBWo4GX8X098Q64/edit#), create accessibility docs for [Accordion](https://vanillaframework.io/docs/patterns/accordion)

**Comment by elioqoshi** [comment]
@bartaz Should we make this part of the UX specs work in the WG since this needs to be done anyway? https://github.com/canonical-web-and-design/vanilla-framework/issues/4123

**Comment by bartaz** [comment]
@elioqoshi We could, but an important part of these docs is related to implementation (what types of elements to use, what `aria-` attributes to use on them, what features JS needs to provide). Do we expect UX people to write those as well?

**Comment by sowasred2012** [comment]
Ideally, UX and dev should both be involved in writing these, or at least if a dev creates the PR, it should be reviewed by a UXer (and vice versa) - one discipline may have considerations that wouldn't occur to the other.

**Comment by bethcollins92** [comment]
First draft [accessibility doc](https://docs.google.com/document/d/1UekmbTagZNVppuiuw3i9aoMpgQBJVdZnSHC2ciW6blY/edit#) for accordions

@bartaz @elioqoshi

**Comment by wgx** [comment]
✅  Done

**Comment by sowasred2012** [comment]
Added a few comments, looking good!

**Comment by bethcollins92** [comment]
Notes are up here: Notes are up here: https://vanillaframework.io/docs/patterns/accordion#accessibility
